docs/monitoring/metrics.md

```diff
@@ -335,6 +345,21 @@ Parameters:
 - `ttl` - time-to-live for transmitted UDP packets
 - `addressingMode` - UDP addressing mode to use (UNICAST/MULTICAST)
 
+Example configuration:
+
+{% highlight yaml %}
+
+metrics.reporters: gang
```

Comment:
If there is no 4 character limitation, I would vote to put the full name here. Definitely personal taste only though. This would also/especially apply to the other abbrvs below for Graphite and others.